charts/gardener/controlplane/values.yaml

```diff
@@ -64,6 +64,11 @@ global:
     insecureSkipTLSVerify: false
     groupPriorityMinimum: 10000
     versionPriority: 20
+    createNewCa: false
+    caFromSecret:
+      enabled: false
+      secretName: mysecret
+    caFromInline: true
     caBundle: |
       -----BEGIN CERTIFICATE-----
       ...
```

Comment:
I don't really like the proposed structure. I would propose something much more simpler, e.g.:

```yaml
caSecretName: ... # (instead of `caFromSecret.secretName`)
caBundle: ...
```

The 2 should be treated as mutually exclusive - if `secretName` is not specified, the `caBundle` is used as now. If `secretName` is specified (by default it should not be specified), the CA should be read from the secret. `createNewCa` should be removed entirely IMO, see my previous comment.